plugins/services/src/js/components/DeclinedOffersTable.js

```diff
@@ -292,11 +293,13 @@ class RejectedOffersTable extends React.Component {
     const {offers} = this.props;
 
     return (
-      <Table className="table table-simple table-header-nowrap table-break-word flush-bottom"
-        colGroup={this.getColGroup()}
-        columns={this.getColumns()}
-        data={offers}
-        sortBy={{prop: 'timestamp', order: 'desc'}} />
+      <MountService.Mount type="DeclinedOffersTable">
```

Comment:
I think this would make sense in this case, but usually I would prefer for more IDs over a generic one as in the future refactoring involves refactoring everywhere in the codebase, rather than just the packages that implement the mounts.